src/docs/cursor-log.dox

```diff
@@ -78,17 +82,20 @@ Here is an example of getting the log cursor's value:
 
 For clarity, imagine a set of three log records:
 
-- the first with a single operation,
-- the second with five operations,
-- the third with a single operation.
+- the first recording an internal operation,
+- the second committing a transaction with five operations,
+- the third committing a transaction with a single operation.
 
-The log cursor's WT_CURSOR::next call will return a total of seven
+The log cursor's WT_CURSOR::next call will return a total of nine
 records.  The first time the log cursor will return a key with a unique
-log ID, a unique transaction ID, and an operation counter of 0.  The
-next five returns from the log cursor will have a common log ID, a
-common transaction ID, and operation counters starting at 1 and ending
-at 5.  The final return from the log cursor will again have a unique log
-ID, a unique transaction ID, and an operation counter of 0.
+log ID, no transaction ID, and an operation counter of 0.  The
+next six returns from the log cursor will have a common log ID, a
+common transaction ID, and operation counters starting at 0, returning
+the whole record and then starting at 1 and ending at 5 for each
+of the five individual operations. The next return from the log cursor
+will again have a unique log ID, a unique transaction ID, and an
+operation counter of 0. And the final return from the log cursor
+will have an operation counter of 1.
```

Comment:
Not sure what "an internal operation" means?

Reply:
I'd add a numbered list, that way each of the elements is explicitly listed, something like:
```
1. log ID=5, no transaction ID, operation counter=0, value of the internal operation
2. log ID=6, transaction ID=30, operation counter=0, all records in transaction 30
3. log ID=6, transaction ID=30, operation counter=1, first record in transaction 30
4. log ID=6, transaction ID=30, operation counter=2, second record in transaction 30
5. log ID=6, transaction ID=30, operation counter=3, third record in transaction 30
6. log ID=6, transaction ID=30, operation counter=4, fourth record in transaction 30
7. log ID=6, transaction ID=30, operation counter=5, fifth record in transaction 30
8. log ID=7, transaction ID=31, operation counter=0, all records in transaction 30
9. log ID=7, transaction ID=31, operation counter=1, first record in transaction 31
```

Reply:
Another thing, is it possible to have `no transaction ID`? Doesn't that translate to a transaction ID of 0?

Reply:
Yes. A transaction id will be 0. I'll note that.